src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd

```diff
@@ -204,4 +205,10 @@
       <xsd:pattern value="(%.+%|true|false)" />
     </xsd:restriction>
   </xsd:simpleType>
+
+  <xsd:simpleType name="boolean_or_star">
+    <xsd:restriction base="xsd:string">
+      <xsd:pattern value="(%.+%|true|false|\*)" />
```

Comment:
This isn't a BC break because it was already not using the the boolean type of XML schema but the locally defined one. I've just copied, pasted and extended the current type definition:

```xml
  <xsd:simpleType name="boolean">
    <xsd:restriction base="xsd:string">
      <xsd:pattern value="(%.+%|true|false)" />
    </xsd:restriction>
  </xsd:simpleType>
```